Apps/Inc/UpdateDisplay.h

```diff
@@ -26,11 +26,12 @@
  * Error types
  */
 typedef enum{
-	UPDATEDISPLAY_ERR_NONE			= 0,
-	UPDATEDISPLAY_ERR_FIFO_PUT		=-1,	// Error putting command in fifo
-	UPDATEDISPLAY_ERR_FIFO_POP		=-2,	// Error popping command from fifo
-	UPDATEDISPLAY_ERR_PARSE_COMP	=-3,	// Error parsing component/val in SetComponent
-	UPDATEDISPLAY_ERR_NO_CHANGE	=-4,		// No change in component value
+	UPDATEDISPLAY_ERR_NONE          =0,
+	UPDATEDISPLAY_ERR_FIFO_PUT      =1,    // Error putting command in fifo
+	UPDATEDISPLAY_ERR_FIFO_POP      =2,    // Error popping command from fifo
+	UPDATEDISPLAY_ERR_PARSE_COMP    =3,    // Error parsing component/val in SetComponent
+	UPDATEDISPLAY_ERR_NO_CHANGE     =4,    // No change in component value
+	UPDATEDISPLAY_ERR_DRIVER        =5     // Driver call returned an error
 } UpdateDisplayError_t;
```

Comment:
Since these just count up, you can just list the names without putting any numbers. It'll look cleaner.

Reply:
Will do, thanks!